source/_posts/2020-08-12-release-114.markdown

```diff
@@ -0,0 +1,1107 @@
+---
+layout: post
+title: "0.114: Beta release notes"
+description: "Beta release notes"
+date: 2020-08-05 00:00:00
+date_formatted: "August 12, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-08-0.114/social.png
+---
+
+<!--<a href='/integrations/#version/0.113'><img src='/images/blog/2020-07-0.113/social.png' style='border: 0;box-shadow: none;'></a>-->
+
+Beta release notes for 0.114.0
+
+Please be sure to report issues on our GitHub issue tracker:
+
+- Issues with core & integrations: <https://github.com/home-assistant/core/issues>
+- Issues with the frontend/UI: <https://github.com/home-assistant/frontend/issues>
+
+Issues introduced in this beta, are generally picked up with priority.
+
+## Introduction
+
+Ooh boy, time flew this release cycle; it is time for 0.114 already!
+
+Personally, I'm hardly even done implementing all automation features added
+in the previous release 😅
+
+Ever since or Lovelace UI started as a beta, back in 2018, one of the most
+requested feature - arguably - is a dark mode. No wonder, as a dark mode is
+added to software everywhere nowadays.
```

Comment:
```suggestion
Ever since our Lovelace UI started as a beta, back in 2018, one of the most
requested features - arguably - is a dark mode. No wonder, as a dark mode is
added to software everywhere nowadays.
```